### PR TITLE
add easyconfig for ABAQUS v6.13.5

### PR DESCRIPTION
**Pull request by boegel**
@wpoely86: please review?

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.13.5-linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name = 'ABAQUS'
+version = '6.13.5-linux-x86_64'
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.13.5-linux-x86_64.eb]: versionsuffix?

File: easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.13.5-linux-x86_64.eb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+homepage = 'http://www.simulia.com/products/abaqus_fea.html'
+description = "Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA."
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%s-%s.tgz' % (name, version.split('-')[0])]
+
+builddependencies = [('Java', '1.7.0_60')]
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.13.5-linux-x86_64.eb]: only build dep?

File: easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.13.5-linux-x86_64.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.13.5-linux-x86_64.eb]: hah, good point, that simplifies the `sources` spec significantly too

File: easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.13.5-linux-x86_64.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.13.5-linux-x86_64.eb]: yes, the install is Java based

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1def691de86e81f813e7 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**